src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,157 @@
+Active mixed content includes scripts, stylesheets, iframes,
+and other code that the browser can download and execute.
+
+### Passive mixed content
+
+Passive mixed content is seen as less problematic yet still poses a security threat to your site and your users.
+For example, an attacker can intercept HTTP requests for images on your site and swap or replace these images;
+the attacker can swap the _save_ and _delete_ button images,
+causing your users to delete content without intending to;
+replace your product diagrams with lewd or pornographic content,
+defacing your site; or replace your product pictures with ads for a different site or product.
+
+Even if the attacker doesn't alter the content of your site,
+an attacker can track users via mixed content requests.
+The attacker can tell which pages a user visits and which products they view based on images or other resources that the browser loads.
+
+The following Glitch demo contains examples of passive mixed content.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/passive-mixed-content?path=index.html&previewSize=100"
+    title="passive-mixed-content on Glitch"
+    allow="encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+If passive mixed content is present most browsers will indicate in the URL bar that the page is not secure,
+even when the page itself was loaded over HTTPS.
+
+Until recently passive mixed content was loaded in all browsers,
+as to block it would have broken many websites.
+This is now beginning to change and so it is vital to update any instances of mixed content on your site.
+
+Chrome is currently rolling out automatic upgrading of passive mixed content where possible.
+Automatic upgrading means that if the asset is available over HTTPS, but has been hardcoded as HTTP,
+the browser will load the HTTPS version. If no secure version can be found the asset will not load.
+
+When mixed content is detected, or auto-upgrading has occurred, this will be noted in the Chrome DevTools console.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="passive-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when mixed content is detected and upgraded">
+</figure>
+
+### Active mixed content
+
+Active mixed content poses a greater threat than passive mixed content.
+An attacker can intercept and rewrite active content,
+thereby taking full control of your page or even your entire website.
+This allows the attacker to change anything about the page,
+including displaying entirely different content,
+stealing user passwords or other login credentials,
+stealing user session cookies,
+or redirecting the user to a different site entirely.
+
+Due to the severity of this threat,
+many browsers already block this type of content by default to protect users,
```

Comment:
Nitpick: I'd change this to most, since all current versions of major browsers block it.